docs/userguide.md

```diff
@@ -527,6 +527,7 @@ spec:
       weight: 1
     "C":
       maxReplicas: 20
+      weight: 1
```

Comment:
Should not the following already covered "C"'s weight?

```
"*":
  weight: 1
```

Reply:
That's not what I find when testing it.

https://github.com/kubernetes-sigs/federation-v2/blob/master/pkg/controller/util/planner/planner.go#L88
"C" exists so it doesn't even consider "*", which means C's weight gets the zero value of an int.

Reply:
We could change it to inherit values from "*", but then what about maxReplicas? Once it's set in "*" there's no way to unset it for any other clusters (other than setting it to 99999 and assuming that's enough).

Reply:
Thanks @tghartland , I see, yes, we need to set `weight` here.